draft-ietf-tls-tls13.md

```diff
@@ -1459,7 +1460,7 @@ non-PSK case discussed in the previous paragraph.
 The server indicates its selected parameters in the ServerHello as
 follows:
 
-- If PSK is being used then the server will send a
+- If PSK is being used, then the server will send a
```

Comment:
I find that the RFC Editor staff have a different idea of when commas are appropriate than I do, so I try to restrain myself from making too many changes of that nature.  (Just FYI; I would probably add this if it was my document.)